CoordImporter/Parser/Parsers.cs

```diff
@@ -67,7 +70,7 @@ public class SirenParser : ITrackerParser
     
     // For the format "(Maybe: Storsie) {LinkChar}Labyrinthos{I1Char} ( 17  , 9.6 ) "
     private static readonly Regex SirenRegex = new Regex(
-        $@"(\(Maybe: (?<mark_name>[\w+ '-]+)\) {LinkChar})?(?<map_name>[\w+ '-]+)(?<instance>[{I1Char}-{I9Char}]?)\s+\(\s*(?<x_coord>[0-9\.]+)\s*,\s*(?<y_coord>[0-9\.]+)\s*\)",
+        $@"\(Maybe: (?<mark_name>[\w+ '-]+)\) {LinkChar}(?<map_name>[\w+ '-]+)(?<instance>[{I1Char}-{I9Char}]?)\s+\(\s*(?<x_coord>[0-9\.]+)\s*,\s*(?<y_coord>[0-9\.]+)\s*\)",
```

Comment:
is this correct? i took off the `?` because we need the link character to identify a message as siren. but is the "Maybe" section actually optional? should the link char just be on the outside of the `?`?

Reply:
This was originally meant to make the siren input be compatible with the train bot. The bot outputs siren-like text (e.g. `(Yilan) Thavnair ( 18.4 , 23.6 )  (Instance ONE)`) but it's explicitly missing the `Maybe: `. I think I must have goofed when tweaking the regex to make it support that format. Having the `Maybe: ` text optional is the important bit

Reply:
aha! that makes sense! i'll add the `?` around `Maybe: ` 👍